GameMaster/cogs/losowe.py

```diff
@@ -30,6 +30,10 @@ async def kostka(self, ctx, k=6, x=1, sep=', '):
         else:
             if k < 4:
                 await ctx.send(embed=error_em('Liczba ścianek musi wynosić co najmniej 4.'))
+            elif k > 100:
+                await ctx.send(embed=error_em('Kostka może posiadać maksymalnie 100 ścian.'))
+            elif x > 100:
+                await ctx.send(embed=error_em('Nie możesz rzucić kościami więcej niż 100 razy!'))
```

Comment:
All info should end the same.

Reply:
Now it looks better

Reply:
But instead of "kościami" should be "kośćmi"
![Screenshot_2020-06-20-16-56-54](https://user-images.githubusercontent.com/39430751/85204804-3f8c4b00-b317-11ea-8d28-1b6ef821cc61.jpg)

Reply:
Now it looks better